src/appleseed/renderer/modeling/bsdf/diffusebtdf.cpp

```diff
@@ -145,6 +145,7 @@ namespace
                 sample.m_value.m_beauty = sample.m_value.m_diffuse;
 
                 sample.m_min_roughness = 1.0f;
+                sample.compute_diffuse_differentials(outgoing);
```

Comment:
Given that we're dealing with transmission here, are the differentials correct? Just wondering about the lack of reflected vs transmitted versions of the diffuse case.

Reply:
Diffuse differentials are always 1/25 of the sphere in the direction of the outgoing vector.
There is no special case for transmitted diffuse differentials.